src/core/core.layoutService.js

```diff
@@ -81,6 +81,15 @@ module.exports = function(Chart) {
 		},
 
 		/**
+		 * Sets (or updates) options on the given `item`.
+		 */
```

Comment:
I updated the PR but also changed a bit the `configure` behavior: only update item properties if there is an option value.